superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -139,8 +139,8 @@ class ExploreResultsButton extends React.PureComponent {
           datasource: `${data.table_id}__table`,
           metrics: [],
           groupby: [],
+          time_range: 'No filter',
```

Comment:
This is a global change. It changes the default time range on the landing from the "Explore" chart button in SQL Lab from "100 years ago" to "No filter". It will effect all visualizations.

Reply:
Oh. Do you want to save this as a separate PR?

Reply:
It is small code change but has separated effect.

Reply:
It is somewhat related though. With the default "100 years ago" filter, applying the fixed range will not look good on the trend line. This change was motivated by the changes to Big Number.

I can create a separate PR nonetheless.